try {
  // code here
  const input = null;

  const solve = () => {
    // code here
  };

  const output = solve();

  console.log(output);
} catch (e) {
  console.error(e);
}
